pom.xml

```diff
@@ -54,11 +59,21 @@
 			<artifactId>h2</artifactId>
 			<scope>runtime</scope>
 		</dependency>
-		<!-- Sending emails -->
 		<dependency>
-			<groupId>io.mailtrap</groupId>
-			<artifactId>mailtrap-java</artifactId>
-			<version>${mailtrap.version}</version>
+			<groupId>com.sap.cloud.db.jdbc</groupId>
+			<artifactId>ngdbc</artifactId>
+			<version>${ngdbc.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.liquibase.ext</groupId>
+			<artifactId>liquibase-hanadb</artifactId>
+			<version>${liquibase.ext.version}</version>
+		</dependency>
+		<!-- Retrying -->
+		<dependency>
+			<groupId>org.springframework.retry</groupId>
+			<artifactId>spring-retry</artifactId>
+			<version>${spring.retry.version}</version>
```

Comment:
spring-retry version can be taken from parent pom, no need to explicitly define it here. Or maybe do you have some reason why it's set explicitly?